src/renderer/scss/component/_file-render.scss

```diff
@@ -9,17 +9,80 @@
   bottom: 0;
   z-index: 1;
   overflow: hidden;
+}
+
+.file-render__viewer {
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  background-color: black;
 
-  .file-render__viewer {
-    margin: 0;
+  iframe,
+  webview {
     width: 100%;
     height: 100%;
-    background: black;
+  }
+}
+
+.document-viewer {
+  overflow: auto;
+  background-color: var(--card-bg);
+  font-size: calc(var(--font-size-subtext-multiple) * 1em);
+}
+
+.document-viewer .markdown-preview {
+  height: 100%;
+  overflow: auto;
+  padding: 32px 16px;
+}
+
+.code-viewer,
+.document-viewer__content {
+  overflow: auto;
+  width: 100%;
+  height: 100%;
+}
+
+/* Code-viewer */
+.code-viewer .CodeMirror {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+  min-height: 100px;
+
+  /* Block native text selection */
+  user-select: none;
+
+  .CodeMirror-code {
+    font-size: 1em;
+    font-weight: 350;
+    line-height: 1.5em;
+    font-family: Menlo, Consolas, 'DejaVu Sans Mono', inconsolata, monospace;
+    letter-spacing: 0.3px;
+    word-spacing: 1px;
+  }
+
+  .CodeMirror-linenumber {
+    color: var(--card-text-color);
+  }
+
+  .CodeMirror .CodeMirror-lines {
+    padding: 4px 0;
+  }
+
+  .CodeMirror-line {
+    padding-left: 16px;
+  }
+
+  .CodeMirror-gutters {
+    border-right: 1px solid var(--color-divider);
+    background: var(--color-bg-alt);
+    padding-right: 8px;
+    z-index: 3;
```

Comment:
Whoops forgot to add this comment. This `z-index` seems a little weird. Is it needed?

Reply:
not sure let me check, probably not :+1: